### DISCUSSION TITLE
Support

### DISCUSSION THREAD

**Discussion by gcristian** [kriomant/ch57x-keyboard-tool#107] [General]
Hey, just want to point out that it works perfectly on a 3x5 buttons with 3 knobs and 3 layers
Wanted to leave that somewhere so you update your support list. I got mine from aliexpress

EDIT: it's actually not working, in the beginning after unplugging the usb cable it held the config but lately lost it, it might be related to how I use my gnome config, or how tweaky I am with everything IDK, I think it's on me, but after careful try and error I started to note that -at least in my case- depending on the layer I'm situated the config get's applied upon it, not respecting the layer nuber, then I realize some layers still have the initial config while other layers have some new config, SO, IDK... I just run out of time to debug and I put it aside.

**Comment by kriomant** [comment]
Strange, as some other people have troubles with it. https://github.com/kriomant/ch57x-keyboard-tool/issues/102

**Comment by SteeluzDH** [comment]
Heyo, just got my 2x3x1 board and trying out my custom binds, but it doesnt seen to want to work. when i try to launch the latest version of the ch57x-keyboard-tool it just shows a window for a sec then nothing. i've tried editing the yaml example file aswell as making my own. any tips?

**Reply by SteeluzDH** [reply]
issue fixed, it was on my end damn defender. Make sure your defender (any firewall) isn't stealth blocking the program!!
